aptos-move/e2e-move-tests/src/lib.rs

```diff
@@ -60,5 +60,10 @@ pub(crate) fn build_package_with_compiler_version(
 ) -> anyhow::Result<BuiltPackage> {
     let mut options = options;
     options.compiler_version = Some(compiler_version);
+    if options.compiler_version.unwrap() != CompilerVersion::V1 {
```

Comment:
Instead of unwrapping the just now set `options.compiler_version`, use `compiler_version` (the param) instead?

Reply:
done